decidim-dev/lib/decidim/dev/test/rspec_support/decidim_sanitization.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.include Decidim::SanitizeHelper
```

Comment:
Can we remove these includes as they're no longer necessary 🙏🏽 ?

```console
$ rg Decidim::SanitizeHelper decidim-*/spec/
decidim-proposals/spec/system/participatory_texts_spec.rb
6:  include Decidim::SanitizeHelper

decidim-initiatives/spec/events/decidim/initiatives/endorse_initiative_event_spec.rb
7:  include Decidim::SanitizeHelper

decidim-meetings/spec/cells/decidim/meetings/meeting_s_cell_spec.rb
8:    include Decidim::SanitizeHelper

decidim-meetings/spec/requests/meeting_search_spec.rb
7:  include Decidim::SanitizeHelper

decidim-meetings/spec/requests/meeting_directory_search_spec.rb
7:  include Decidim::SanitizeHelper

decidim-meetings/spec/system/admin/admin_manages_meetings_copy_spec.rb
13:  include Decidim::SanitizeHelper

decidim-meetings/spec/system/admin/admin_manages_meetings_spec.rb
16:  include Decidim::SanitizeHelper

decidim-initiatives/spec/mailers/decidim/initiatives/initiatives_mailer_spec.rb
9:      include Decidim::SanitizeHelper

decidim-initiatives/spec/cells/decidim/initiatives_votes/vote_cell_spec.rb
8:  include Decidim::SanitizeHelper

decidim-initiatives/spec/requests/initiative_search_spec.rb
6:  include Decidim::SanitizeHelper

decidim-accountability/spec/requests/result_search_spec.rb
7:  include Decidim::SanitizeHelper

decidim-admin/spec/system/admin_manages_organization_admins_spec.rb
6:  include Decidim::SanitizeHelper

decidim-admin/spec/system/admin_manages_organization_areas_spec.rb
6:  include Decidim::SanitizeHelper

decidim-admin/spec/system/admin_manages_organization_scopes_spec.rb
6:  include Decidim::SanitizeHelper

decidim-assemblies/spec/system/admin/admin_manages_assemblies_types_spec.rb
6:  include Decidim::SanitizeHelper

decidim-accountability/spec/controllers/decidim/accountability/admin/results_controller_spec.rb
10:        include Decidim::SanitizeHelper

decidim-debates/spec/events/decidim/debates/create_debate_event_spec.rb
6:  include Decidim::SanitizeHelper
```

Reply:
Let's merge it as it is, and i will tackle it in a separate PR. i would really want to keep the PRs under 50 files changed.

Reply:
Ok, lets do it in another. As far as I remember there's no problem including a module something 2 times

Reply:
Keeping track of this in #12336

Reply:
Done in the #12337 PR. The rebase will be done after we merge this.